modules/configure-acs-email.adoc

```diff
@@ -2,10 +2,15 @@
 //
 // * integration/integrate-using-email.adoc
 :_mod-docs-content-type: PROCEDURE
-[id="configure-acs-kube-security-platform_{context}"]
-= Configuring the email plugin 
+[id="configure-acs-email_{context}"]
+= Configuring the email plugin
 
-The {product-title-short} notifier can send email to a recipient specified in the integration, or it can use annotations to determine the recipient. 
+The {product-title-short} notifier can send email to a recipient specified in the integration, or it can use annotations to determine the recipient.
+
+[IMPORTANT]
+====
+If you are using {product-title-managed-short}, it blocks port number `25` by default. Configure your mail server to use port `587` or `465` to send email notifications.
+====
 
 To use an annotation to dynamically determine an email recipient:
```

Comment:
I know this is existing, but can you make this lead-in to the list a complete sentence? Alternatively, could the information here be moved down to step 11 where it seems to be relevant?